osu.Game/Skinning/SkinnableSprite.cs

```diff
@@ -65,7 +65,7 @@ protected override Drawable CreateDefault(ISkinComponent component)
 
         public bool UsesFixedAnchor { get; set; }
 
-        private class SpriteComponent : ISkinComponent
+        public class SpriteComponent : ISkinComponent
```

Comment:
Dunno about this accessibility change, how about `internal` instead?

Reply:
Sure, seems fine.